node/engine/chainservice/eth_chainservice.go

```diff
@@ -65,6 +74,15 @@ const MAX_QUERY_BLOCK_RANGE = 2000
 // This has been reduced to 15 seconds to support local devnets with much shorter timeouts.
 const RESUB_INTERVAL = 15 * time.Second
 
+// REQUIRED_BLOCK_CONFIRMATIONS is how many blocks must be mined before an emitted event is processed
+const REQUIRED_BLOCK_CONFIRMATIONS = 0
```

Comment:
This need to be set to something > 0 to work against the Filecoin Calibration testnet. However, it needs to be set to 0 to pass tests that run against the simulated chain used in testing. In a follow-up PR we will address this by adding logic to set the value based on the chainId (i.e. against simulated test chains we will use 0 and against all other chainIds we will use 2)